src/main/java/ch/wisv/areafiftylan/dto/TransferDTO.java

```diff
@@ -0,0 +1,14 @@
+package ch.wisv.areafiftylan.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by beer on 7-1-16.
+ */
+public class TransferDTO {
```

Comment:
On a side note: It doesn't do the same thing. If replaced with string the entire JSON string will be injected into the goalUsername field and not just the corresponding field of the JSON object.